hugr-model/src/v0/text/print.rs

```diff
@@ -178,9 +170,8 @@ impl<'p, 'a: 'p> PrintContext<'p, 'a> {
                     this.print_text(decl.name);
                 });
 
-                for param in decl.params {
-                    this.print_param(*param)?;
-                }
+                this.print_params(decl.params)?;
```

Comment:
If we are sure we want constraints in Alias too (I argue against elsewhere), then it is true that everywhere you introduce params you also introduce constraints; maybe a `struct ParamsAndConstraints<'a>` (maybe `ConstrainedParams`?) would be useful, as it'd allow combining all these `print_params(...); print_constraints(...)`

Reply:
I agree that we'd probably want to avoid constraints on aliases for now.